### PR TITLE
Loopback transform directly returns the message

### LINKED ISSUE #472: Rewrite the loopback transform to directly return the message
Rewrite the loopback transform such that it just directly returns the messages without processing.
Will also need to remove it from the public API as this will perform poorly in real world scenarios.
(take from https://github.com/shotover/shotover-proxy/pull/453)

### PR DESCRIPTION
**Pull request by conorbros**
closes #472

### CODE REVIEW

File: shotover-proxy/src/transforms/protect/mod.rs [hunk 1]
```diff
@@ -269,7 +269,9 @@ mod protect_transform_tests {
     use crate::transforms::protect::ProtectConfig;
     use crate::transforms::{Transform, Transforms, Wrapper};
 
-    #[tokio::test(flavor = "multi_thread")]
+    //#[tokio::test(flavor = "multi_thread")]
+    #[allow(unused)]
+    #[ignore]
```
Comment (rukai) [inline shotover-proxy/src/transforms/protect/mod.rs:274]: Can you remove these changes please?
Dont want merge conflicts with https://github.com/shotover/shotover-proxy/pull/484

Comment (conorbros) [reply shotover-proxy/src/transforms/protect/mod.rs:274]: I'll rebase now that the test is fixed.

**Review by rukai — APPROVED** [review]
lgtm

**Comment by conorbros** [comment]
Hmm, it's failing on the new protect tests.

File: shotover-proxy/src/transforms/mod.rs [hunk 2]
```diff
@@ -105,6 +107,7 @@ impl Transforms {
             Transforms::Tee(m) => m.transform(message_wrapper).await,
             Transforms::DebugPrinter(p) => p.transform(message_wrapper).await,
             Transforms::Null(n) => n.transform(message_wrapper).await,
+            #[cfg(test)]
```
Comment (rukai) [inline shotover-proxy/src/transforms/mod.rs:110]: Oh wait, cant we get away with just adding the `#[cfg(test)]` to the `TransformsConfig::get_transforms` Loopback arm?

Comment (conorbros) [reply shotover-proxy/src/transforms/mod.rs:110]: Yes. Though only compiling/including what we need in our build would be a good habit to follow.

Comment (rukai) [reply shotover-proxy/src/transforms/mod.rs:110]: Is the advantage of that a smaller binary size?
I think in general we want to keep our test and production binaries as close as possible.
But in this particular case im not super worried either way, so just resolve this if you'd rather leave it as is.

Comment (conorbros) [reply shotover-proxy/src/transforms/mod.rs:110]: Yes I was thinking smaller binary size.

**Review by benbromhead — APPROVED** [review]

**Pull request merged**